### DISCUSSION TITLE
OpenShell Version 4.4.163

### DISCUSSION THREAD

**Discussion by MGBLN39** [Open-Shell/Open-Shell-Menu#683] [Q&A]
I am using OpenShell version 4.4.163 under Windows 10 Prof. 64-bit.
There was a time when you could move the entries in the Open Shell / Window stratmenu, whether folders or shortcuts, to any location using drag & drop. Unfortunately this possibility was lost with one of the updates. If you try it anyway, you create an indescribable chaos in all programs. Because every folder that you cross over with the folder or link that you have checked with the mouse opens immediately. After you have run over more than three such entries you lose any overview.
Why doesn't it work like it used to ?????
![Bild 016](https://user-images.githubusercontent.com/78050972/113512590-a9ec4000-9565-11eb-8874-caf6f23b7d5f.jpg)
Thank you for any useful answer.

**Comment by bonzibudd** [comment]
Try changing the "Menu delay" in Open-Shell settings, or try one of the other options here:

![image](https://user-images.githubusercontent.com/61938331/116009082-8c028000-a5e5-11eb-8673-b1af65ad7c9f.png)

This seems to match up to the behavior you are describing.

If that doesn't work, try right-clicking an option in the programs list and toggling "Auto Arrange."

**Comment by NebelNidas** [comment]
Moving the different folders or files via drag-and-drop works perfectly fine for me 🤷‍♂️
